cas-airflow/values.yaml

```diff
@@ -1,33 +1,90 @@
 airflow:
+  web:
+    initialDelaySeconds: 60
+  dags:
+    git:
+      url: https://github.com/bcgov/cas-airflow-dags.git
+      ref: master
+    initContainer:
+      enabled: true
+      installRequirements: false
   airflow:
     image:
       repository: docker-registry.default.svc:5000/wksv3k-tools/cas-airflow
-      tag: 0.0.1
+      tag: '0.1.0-pre.033'
+    executor: Kubernetes
+    config:
+      AIRFLOW__CORE__REMOTE_LOGGING: True
+      AIRFLOW__CORE__REMOTE_BASE_LOG_FOLDER: gs://wksv3k-tools-app1
+      AIRFLOW__CORE__REMOTE_LOG_CONN_ID: gcs_logs
+      # AIRFLOW__KUBERNETES__GIT_REPO: https://github.com/bcgov/cas-airflow-dags.git
+      # AIRFLOW__KUBERNETES__GIT_BRANCH: master
+      # AIRFLOW__KUBERNETES__GIT_DAGS_FOLDER_MOUNT_POINT: /usr/local/airflow/dags
+      AIRFLOW__KUBERNETES__KUBE_CLIENT_REQUEST_ARGS: '{\"_request_timeout\":[300,300]}'
+      # see https://github.com/puckel/docker-airflow/issues/521
+      AIRFLOW__KUBERNETES__LOGS_VOLUME_CLAIM: cas-airflow-logs
+      AIRFLOW__KUBERNETES__DAGS_VOLUME_CLAIM: cas-airflow
+      AIRFLOW__KUBERNETES__NAMESPACE: wksv3k-tools
+      AIRFLOW__KUBERNETES__WORKER_SERVICE_ACCOUNT_NAME: cas-airflow
+      # AIRFLOW__KUBERNETES__WORKER_RUN_AS_USER: '1000880000'
+      # AIRFLOW__KUBERNETES__WORKER_FS_GROUP: '1000880000'
+      # see https://github.com/helm/helm/issues/1707
```

Comment:
I wanted to leave this in as a reminder that we couldn't get this working because of a golang casting error. Does the issue reference convey that?

Reply:
I don't think I would make the connection between them without more context